src/app/design/guidelines/back-button/index.html

```diff
@@ -0,0 +1,165 @@
+  </p>
+
+  <stache-page-anchor>
+    Tabs as containers for viewing items
+  </stache-page-anchor>
+
+  <p>
+    If tabs contain removable items such as lists or reports, don't include each tab in the browser history.
+  </p>
+
+	<div class="screenshot screenshot-info"> 
+    <img src="~/assets/img/guidelines/back-button/tabcontainers-backbutton.png" alt="Tabs as page navigation" />
+	</div>
+
+<br>
+
+  <sky-row>
+    <sky-column>
+      <img class="screenshot screenshot-info" src="~/assets/img/guidelines/back-button/verticaltabs-backbutton.png" alt="Vertical tabs as navigation" />
+    </sky-column>
+  </sky-row>
+
+  <div class="sky-subsection-heading" style="margin-top: 20px;">Focus</div>
+
+  <p>
+    Based on guidance from <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel">WAI-ARIA Authoring Practices for tabs</a>, maintain focus on activated tabs.
+  </p>
+
+  <div class="sky-subsection-heading">New page announcement for screenreaders</div>
+
+  <p>
+    This option doesn't require new page announcements. The selected tab receives an updated <sky-code>aria-selected="true"</sky-code> property which is communicated to assistive technologies.
+  </p>
+
+  <stache-page-anchor>
+    Modals (all sizes)
+  </stache-page-anchor>
+
+  <p>
+    When users open a modal from a SKY UX SPA, they should be able to use the back button to close the modal and return to the original context.
+  </p>
+
+  <div class="sky-subsection-heading">Avoid loss of work</div>
+
+  <p>
+    If a user selects the back button on a form with unsaved data, the SKY UX application asks the user if they want to close the form without saving their data or work using the confirmation dialog.
+  </p>
+
+  <div class="sky-subsection-heading">Focus</div>
+
+  <p>
+    In all circumstances, focus moves to an element contained in the dialog. Set the initial focus on the first focusable element unless a condition where doing otherwise is advisable.
+  </p>
+
+  <p>
+    If focus on the first interactive element causes the content to scroll out of view, add <sky-code>tabindex=-1</sky-code> to a static element at the top of a dialog. For example, add the code to a dialog title or the first paragraph to initially focus the element. For more best practices, see <a href="https://www.w3.org/TR/wai-aria-practices-1.1/#dialog_modal">W3C's WAI-ARIA Authoring Practices for Dialog (Modal)</a>.
+  </p>
+
+  <div class="sky-subsection-heading">New page annoucement for screenreaders</div>
+
+  <p>
+    This option doesn't require new page announcements.
+  </p>
+
+  <stache-page-anchor>
+    Maintaining context
+  </stache-page-anchor>
+
+  <p>
+    For inline interactions, determine the persistence level that makes sense for the change of state the user initiated. When users select the back button, return them to the context they left and don’t make them re-orient, and in some cases re-interact with the system, to return to where they were.
```

Comment:
I'd change "the user initiated" to "that users initiate." ... Also, the second sentence is too long and it needs to be read a second time to read the stuff after "and." I'd make it a separate sentence: ".... left. Don’t make them re-orient, and in some cases re-interact with the system, to return to that context."